dspace-oai/src/main/java/org/dspace/xoai/app/XOAI.java

```diff
@@ -714,4 +725,54 @@ private static void usage() {
         }
     }
 
+    /**
+     * Method to cross references handles of items that have been deleted with OAI core docs of said handle.
+     * If a match is found update that document.
+     * If no match is found create a document for persistence sake
+     *
+     * @param solrServer
+     *            The configured solr server to commit to
+     * @param handle
+     *            The handle to cross reference
+     *
+     */
+    private void updateDocsWithDeletes(SolrClient solrServer, String handle) {
+        try {
+            SolrQuery params = new SolrQuery("item.handle:" + handle);
+            SolrDocument result = SolrSearchQuerySingleSupportNull(solrServer, params);
+            if (result != null) {
+                solrServer.deleteByQuery(params.getQuery());
+            }
+            SolrInputDocument document = new SolrInputDocument();
+            document.setField("item.handle", handle);
+            document.setField("item.deleted", true);
+            document.setField("item.public", true);
+            document.setField("item.lastmodified ", new Date());
```

Comment:
There's an extra space after "item.lastmodified" here that needs to be removed (with the extra space an error occurs whenever you attempt to reindex).

Reply:
@tdonohue Sorry for the typo. Corrected: https://github.com/DSpace/DSpace/pull/2856/commits/0ae0d846365777cd2e642bba510910d8fcb4258c

I will also see if I can reproduce this issue early next week and take a stab at it.